src/components/Layouts.jsx/MainLayout.jsx

```
@@ -702,48 +730,183 @@ const MainLayout = (props) => {
                             >
-                              {percentage <= 0 ? 0 : percentage}/100
-                            </span>
-                            <br />
+                              {(props.pageName === "wordsorimage" || "m5") &&
+                                storedData?.map((elem, i) => (
+                                  <Stack
+                                    justifyContent={"start"}
+                                    alignItems={"center"}
+                                    direction={"row"}
+                                    mt={i > 0 && "25px"}
+                                  >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)